server/mediaserver_test.go

```diff
@@ -91,6 +88,19 @@ func (d *stubDiscovery) GetOrchestrators(num int) ([]*net.OrchestratorInfo, erro
 	return d.infos, err
 }
 
+func (d *stubDiscovery) SuspendOrchestrator(orch string) {
```

Comment:
Doesn't look like this method is used.

Reply:
fixed in 2c78fb9a784ea81c42a9bdbd0d91c71ab6fc255b